### PR TITLE
[Release] React-Native sample update with UI Library 1.6.0 API

### PR DESCRIPTION
**Pull request by Mohtasim**
## Purpose

* React-Native sample update with UI Library 1.6.0 API

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: AzureCommunicationUIDemoApp/android/app/build.gradle [hunk 1]
```diff
@@ -200,7 +200,11 @@ dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:+"  // From node_modules
     
+<<<<<<< HEAD
```
Comment (iaulakh) [inline AzureCommunicationUIDemoApp/android/app/build.gradle:203]: Please resolve

File: AzureCommunicationUIDemoApp/ios/Podfile [hunk 2]
```diff
@@ -11,29 +11,13 @@ require Pod::Executable.execute_command('node', ['-p',
...
-#     ...(process.env.NO_FLIPPER ? { 'react-native-flipper': { platforms: { ios: null } } } : {}),
-# ```
-flipper_config = ENV['NO_FLIPPER'] == "1" ? FlipperConfiguration.disabled : FlipperConfiguration.enabled
-
-linkage = ENV['USE_FRAMEWORKS']
-if linkage != nil
-  Pod::UI.puts "Configuring Pod with #{linkage}ally linked Frameworks".green
-  use_frameworks! :linkage => linkage.to_sym
-end
+source 'git@github.com:iaulakh/calling_ui_library_private_drop.git'
```
Comment (iaulakh) [inline AzureCommunicationUIDemoApp/ios/Podfile:14]: this can be removed now as packages are published

File: AzureCommunicationUIDemoApp/App.js [hunk 3]
```diff
@@ -265,7 +268,7 @@ const App = () => {
               </View>
               <View style={styles.settingsSectionContainerView}>
                 <Text style={styles.settingsHeaderText}>Localization</Text>
-                <RNPickerSelect
+                {/* <RNPickerSelect
```
Comment (iaulakh) [inline AzureCommunicationUIDemoApp/App.js:271]: is this package still causing issue? can we try to update and see if it works?

Comment (Mohtasim) [reply AzureCommunicationUIDemoApp/App.js:271]: Yes. I tried with different version. The picker is making build error due to underlaying version missmatch. I also removed it from the package dependency. 
We can update UI with stable components to select assets when we take on the UI upgrade work.

File: AzureCommunicationUIDemoApp/ios/Podfile (hunk 2 above)
Comment (Mohtasim) [reply AzureCommunicationUIDemoApp/ios/Podfile]: Thanks. Removed.

File: AzureCommunicationUIDemoApp/android/app/build.gradle (hunk 1 above)
Comment (Mohtasim) [reply AzureCommunicationUIDemoApp/android/app/build.gradle]: Thanks. Resolved.

File: AzureCommunicationUIDemoApp/App.js (hunk 3 above)
Comment (Mohtasim) [reply AzureCommunicationUIDemoApp/App.js]: @iaulakh With upgrade to 0.72 of React-Native, the issue for picker has been resolved. Please check to validate for the release.

**Review by ahammer — APPROVED** [review]
Tested and validated on Mac (iOS and Android) and Windows (Android) and lgtm

**Review by iaulakh — APPROVED** [review]

**Pull request merged**